magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -175,65 +175,60 @@ event_classifier:
-        'intensity_width_2',
-        'h_max',
-        'impact',
+        "intensity",
+        "length",
+        "width",
+        "skewness",
+        "kurtosis",
+        "slope",
+        "intensity_width_2",
+        "h_max",
+        "impact",
     ]
 
     gamma_offaxis:
-        min: 0.2   # unit: [deg]
-        max: 0.6   # unit: [deg]
+        min: null
+        max: null
 
 
 create_irf:
     quality_cuts: null
-    irf_type: 'software'   # select 'hardware', 'software', 'software_with_any2' or 'magic_stereo'
-    dl2_weight: null   # type of the weight for averaging tel-wise DL2 parameters, null, 'var' or 'intensity'
-    obs_time: 50   # used for weighting background events, unit: [hour]
+    irf_type: "software"  # select "software(_only_3tel)", "magic_only" or "hardware"
+    dl2_weight_type: "simple"  # select "simple", "variance" or "intensity"
+    irf_obs_time: "50 h"  # used for creating a background HDU
 
-    energy_bins:   # log space
-        start: 0.01   # unit: [TeV]
-        stop: 1000   # unit: [TeV]
-        num: 26   # number of edges (nbins = num - 1)
+    energy_bins:  # log space
+        start: "0.01 TeV"
+        stop: "1000 TeV"
+        n_edges: 26
 
-    migration_bins:   # log space
+    migration_bins:  # log space
         start: 0.2
         stop: 5
-        num: 31   # number of edges (nbins = num - 1)
+        n_edges: 31
 
-    fov_offset_bins:  # linear space
-        start: 0.3   # unit: [deg]
-        stop: 0.5   # unit: [deg]
-        num: 2   # number of edges (nbins = num - 1)
-
-    bkg_fov_offset_bins:   # linear space
-        start: 0  # unit: [deg]
-        stop: 10  # unit: [deg]
-        num: 21  # number of bins in linear space
+    bkg_fov_offset_bins:  # linear space
+        start: "0 deg"
+        stop: "10 deg"
+        n_edges: 21
 
     gammaness:
-        cut_type: 'dynamic'   # select 'global' or 'dynamic'
-        global_cut_value: 0.8   # used for the global cut
-        efficiency: 0.9   # used for the dynamic cut
-        min_cut: 0.05   # used for the dynamic cut
-        max_cut: 0.85   # used for the dynamic cut
+        cut_type: "dynamic"  # select "global" or "dynamic"
+        global_cut_value: 0.8  # used for the global cut
+        efficiency: 0.9  # used for the dynamic cuts
+        min_cut: 0.05  # used for the dynamic cuts
+        max_cut: 0.85  # used for the dynamic cuts
 
     theta:
-        cut_type: 'global'   # select 'global' or 'dynamic'
-        global_cut_value: 0.2   # used for the global cut
-        efficiency: 0.75   # used for the dynamic cut
-        min_cut: 0.1   # used for the dynamic cut
-        max_cut: 0.45   # used for the dynamic cut
+        cut_type: "dynamic"  # select "global" or "dynamic"
+        global_cut_value: "0.2 deg"  # used for the global cut
+        efficiency: 0.75  # used for the dynamic cuts
+        min_cut: "0.1 deg"  # used for the dynamic cuts
+        max_cut: "0.45 deg"  # used for the dynamic cuts
```

Comment:
0.45 deg is actually a lot, there is an overlap between ON and OFF region

Reply:
I agree, actually I also thought the same thing. Now the `max_cut` is updated to 0.3 deg.

Reply:
Please let me know if you think that 0.3 deg is OK or not.